src/app/pages/login/login.component.ts

```diff
@@ -40,8 +43,10 @@ export class LoginComponent implements OnInit {
             role: 'admin' //change when we have user collection
           });
         this.router.navigate(['/home']);
+        this.spinnerService.close();
       }).catch(err => {
         this.errorlogin = true;
+        this.spinnerService.close();
       });
```

Comment:
Instead to put two times   `this.spinnerService.close();` use finally

Reply:
```suggestion
      }).catch(err => {
        this.errorlogin = true;
      }).finally(()=>{  this.spinnerService.close();});
```

Reply:
done